app/views/question_flow/forms/_outgoings.html.slim

```diff
@@ -1,28 +1,51 @@
 - partner ||= false
+- conditionals = FeatureFlags.enabled?(:conditional_reveals, @check.session_data)
 = form_for(@form, url: request.path, method: :put) do |form|
   = form.govuk_error_summary t("generic.error_summary_title")
   = render "shared/heading",
            header_text: t("question_flow.#{i18n_key}.heading"),
-           post_header_text: t("question_flow.#{i18n_key}.second_caption"),
+           post_header_text: t("question_flow.#{i18n_key}.#{'second_' unless conditionals}caption"),
            tag: (:partner if partner)
   - OutgoingsForm::PAYMENT_TYPES.each do |type|
     - next if type == :childcare_payments && !@form.eligible_for_childcare_costs?
-    h2.govuk-heading-m class="govuk-!-margin-bottom-2"
-      = t("question_flow.#{i18n_key}.#{type}.label")
-    - t("question_flow.#{i18n_key}.#{type}.hints").each do |hint|
-      span.govuk-caption-m class="govuk-!-margin-bottom-4"
-        = hint
 
-    = render "shared/money_input",
-            form:,
-            field: :"#{type}_value",
-            width: 10,
-            label_text: t("question_flow.#{i18n_key}.enter_amount"),
-            set_aria: true
-    = form.govuk_collection_radio_buttons :"#{type}_frequency",
-                                          @form.frequencies,
-                                          :first,
-                                          :last,
-                                          legend: -> { tag.p(t("question_flow.#{i18n_key}.enter_frequency")) }
+    - if FeatureFlags.enabled?(:conditional_reveals, @check.session_data)
```

Comment:
question - why not use the `conditionals` variable you created?

Reply:
Because I'm a muppet. Will fix.